docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/AveragedPerceptronWithOptions.cs

```diff
@@ -33,23 +35,30 @@ public static void Example()
             };
 
             // Define the trainer.
-            var pipeline = mlContext.BinaryClassification.Trainers.AveragedPerceptron(options);
+            var pipeline = mlContext.BinaryClassification.Trainers
+                .AveragedPerceptron(options);
+
```

Comment:
extra line

Reply:
yeah noticed that a couple minutes ago...looking for the tt/ttinclude file that is probably causing it